ftest.sh

```diff
@@ -304,6 +304,16 @@ if ${SETUP_ONLY:-false}; then
     exit 0
 fi
 
+# check if slurm needs to be configured for soak
+if [[ ${TEST_TAG_ARG} =~ "soak" ]]; then
```

Comment:
(lint) This is actually an end quote, but due to next char it looks suspect. [SC1079]
(lint) The double quotes around this do nothing. Remove or escape them. [SC2140]